README.md

```diff
@@ -1,4 +1,4 @@
-# ai2thor-experiments
+# cups-rl
```

Comment:
Change title to `cups-rl -- Customizable Unity Photorrealistic Simulation for Reinforcement Learning algorithms`  or something like that. If not, mention it somewhere in the intro

Reply:
done